### PR TITLE
Make VPN client runnable without root

### LINKED ISSUE #382: Avoid running VPN as root
**Feature description**

Currently we need to run `VPN-client` and `VPN-server` as root in order to successfully create the TUN interfaces. It would be preferable to create a new user with access to only those directories and parts of the system needed to create the TUN interface. 

**Is your feature request related to a problem? Please describe.**
Skywire VPN needs to run as root currently.

### PR DESCRIPTION
**Pull request by Darkren**
Did you run `make format && make check`?
Yes
Fixes #382 

 Changes:	
- VPN client now is runnable by non-root users. But, the binary has to be owned by root and have permissions like 4755, so any user could run it
- Linux capabilities are used for Linux version.

How to test this PR:
`Mac OS`:
1. `make bin`
2. `make host-apps`
3. `sudo chown root ./apps/vpn-client`
4. `sudo chmod 4755 ./apps/vpn-client`
5. `./skywire-visor ./skywire-config.json`

`Linux`:
1. `make bin`
2. `make host-apps`
3. `sudo setcap 'cap_net_admin+p' ./apps/vpn-client`
4. `./skywire-visor ./skywire-config.json`

### CODE REVIEW

File: internal/vpn/os_linux.go [hunk 1]
```diff
@@ -42,3 +43,35 @@ func AddRoute(ip, gateway string) error {
 func DeleteRoute(ip, gateway string) error {
 	return run("ip", "r", "del", ip, "via", gateway)
 }
+
+var (
```
Comment (nkryuchkov) [inline internal/vpn/os_linux.go]: Could we get rid of this block and return errors from `syscall` directly? I think this doesn't improve readability.

File: internal/vpn/os_linux.go [hunk 2]
```diff
@@ -42,3 +43,35 @@ func AddRoute(ip, gateway string) error {
...
+		return errEINVAL
+	case syscall.ENOENT:
+		return errENOENT
+	}
+	return e
+}
+
+// Setuid sets uid of current OS user.
+func Setuid(uid int) (err error) {
+	// due to iss. 1435 `syscall.Setuid` is disabled for Linux system, so we'
```
Comment (nkryuchkov) [inline internal/vpn/os_linux.go]: Could you please add the link (https://github.com/golang/go/issues/1435) to comment? That could be a bit clearer. Also, `so we'` looks unfinished.

Comment (Darkren) [reply internal/vpn/os_linux.go]: Not used anymore

File: internal/vpn/os_linux.go (hunk 1 above)
Comment (Darkren) [reply internal/vpn/os_linux.go]: Not used anymore

**Review by nkryuchkov — APPROVED** [review]

**Review by jdknives — APPROVED** [review]

**Pull request merged**